azure-pipelines.yml

```diff
@@ -138,14 +138,14 @@ stages:
               source activate qiskit-aer-$(Build.BuildNumber)
               conda install -c conda-forge --yes --quiet --name qiskit-aer-$(Build.BuildNumber) python=$(python.version) osqp
               pip install -r requirements-dev.txt
-              pip install git+https://github.com/Qiskit/qiskit-terra.git
+              pip install qiskit-terra
```

Comment:
Shouldn't we put conditionals depending on the branch here as well?

Reply:
oops, yep I completely forgot about  azure too. I'll update this now

Reply:
Done in: https://github.com/Qiskit/qiskit-aer/pull/633/commits/801d6293429e1ea9865da2a4ba651d240693c453